tools/install.py

```
@@ -165,14 +165,17 @@ def nsolid_cli_files(options, action):
 def nsolid_strict_files(options, action):
   target_path = 'lib/node_modules/ncm-ng/'
 
-  for dirname, subdirs, basenames in os.walk('deps/ncm-ng', topdown=True):
+  def safe_action(paths, destination):
     try:
-      subdirs[:] = filter('test'.__ne__, subdirs) # skip test suites
-      paths = [os.path.join(dirname, basename) for basename in basenames]
-      action(options, paths, target_path + dirname[11:] + '/')
+      action(options, paths, destination)
     except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)